### DISCUSSION TITLE
GNU Image Manipulation Program (GIMP)

### DISCUSSION THREAD

**Discussion by PoorPocketsMcNewHold** [catppuccin/catppuccin#2023] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

![catppuccinoattempt](https://user-images.githubusercontent.com/13170204/233839672-e92e1cb8-4170-4dbb-99e9-a5164e30ae58.jpg)
![catppuccinoattempt](https://user-images.githubusercontent.com/13170204/233839815-a4908351-f1f5-4c08-8cad-e3ed156e8455.jpg)
(With the Lavender color accent)
(Edited from the [PhotoGIMP](https://github.com/Diolinux/PhotoGIMP) theme and setup. I don’t have any prototypes of the colors. I mainly edited it on top of it.)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by rubyowo** [comment]
I mean, GIMP uses the GTK theme so...

**Reply by isabelroses** [reply]
GTK even used to stand for Gimp Tool Kit, until Gnome took over and called it Gnome Tool Kit. So have to agree with ruby.

But I mean it couldn't hurt to have an individual theme for GIMP, incase people are avoiding GTK.

**Reply by PoorPocketsMcNewHold** [reply]
@rubyowo ![image](https://user-images.githubusercontent.com/13170204/233839949-4085b1d1-6223-4828-bb3a-8d543a44aba8.png)
 I believe you see the actual intend into asking for a proper port, instead of waiting for better GTK support, when it’s still having issues switching everything from GTK2 to GTK3, and don’t talk about LibAdwaita/GTK4 with it.

**Reply by rubyowo** [reply]
![image](https://user-images.githubusercontent.com/105302757/233840107-24c23971-18b7-46d3-89b4-1fe70e16a0c7.png)
works for me

**Reply by spookyintheam** [reply]
Also adding to this, GIMP recently finished it's conversion to GTK 3 so once it rolls out, GTK themes can better support GIMP.

**Reply by PoorPocketsMcNewHold** [reply]
My bad. Indeed, got a previous gtkrc override file in it.
![image](https://user-images.githubusercontent.com/13170204/233840991-76fe96f9-0537-4db4-b05f-2c6b4fb23c89.png)
But i feel either way, that there’s some advantage into making it custom for some elements to better match with other ports.
